db/migrate/20230118185730_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
```

Comment:
- [ ] Kindly make sure to use `text` for `Bio` instead of `string` so your migrations are created based on the `ERD`

Reply:
Updated. Thank you

Reply:
Updated. Thank you